11ty/definitions/master-slave.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Master Slave
+slug: master-slave
+defined: false
+speech: noun
+alt_words:
+  - Agency & Operatives
+  - Leader & Follower
+  - Main & Secondary
+  - Parent & Child, Helpers, or Workers
+  - Primary & Replica
+  - Primary & Secondary
+  - Trunk & Branch
+reading:
+  - text: Rename Your Git Branch From Master To Main
+    href: https://www.hanselman.com/blog/EasilyRenameYourGitDefaultBranchFromMasterToMain.aspx
+---
+
+A model of asymmetric communication or control where one device or process have total control over other components or are controlled by a component.
+
+## Issues
+
+Master/slave terminology is problematic given that slavery was institutionalised in the United States for hundreds of years. Yet, this type of offensive language is still used casually in many programming languages.
```

Comment:
```suggestion
Master/slave terminology is problematic because it serves as a reminder that slavery was and continues to be a global institutionalised practice. Yet, this type of offensive language is still used casually in many programming languages.
```